### DISCUSSION TITLE
Github Desktop not syncing

### DISCUSSION THREAD

**Discussion by wigdaddy72** [community/community#27842] [Other Feature Feedback, Questions, & Ideas]
How do I get everything that is on desktop to sync to online?  All I see is the Repository itself but none of the files or codes are online

**Comment by jfullstackdev** [comment]
are you connected to the Internet that time ? can you send us the screenshots of that so we can take a closer look

**Comment by wigdaddy72** [comment]
I figured it out, the codes weren't committed yet, once I did that I was able to publish online.

Sorry, newbie mistake, still trying to learn

Thanks!
________________________________
